psalm-report.json

```diff
@@ -0,0 +1,210 @@
+{
+    "error": [],
+    "warning": [],
+    "deprecation": [
+        {
+            "severity": "info",
+            "line_from": 43,
+            "line_to": 43,
+            "type": "DeprecatedClass",
+            "message": "Class Spryker\\Glue\\GlueApplication\\Rest\\Request\\Data\\RestRequestInterface is marked as deprecated",
```

Comment:
This module implements the deprecated API concept, was that planned or agreed with someone?

Reply:
Please re-generate the report, deprecations should go.